### DISCUSSION TITLE
Game crashes when trying to save

### DISCUSSION THREAD

**Discussion by ghost** [voxeltycoon/issues#677] [Troubleshooting] [answered]
**Describe the bug**
The game keeps crashing (showing bug report) when trying to save. Please find log file below:

**Steps to reproduce**
1. Load the saved game / continue the saved game from main menu
2. After loading, save the game
3. Crash / Bug report shows

**Build version**
0.85.1.0 Steam

**Platform**
Mac OS, Big Sur 11.3

**Reproducibility**
Happens continuously in the (save)game at the point where I am now. Keeps happening when loading the saved game. I didn't run into this issue before.

**Logs/saves**
[Player.log](https://github.com/voxeltycoon/issues/files/6409003/Player.log)

**Comment by andrewpey** [comment]
Seems to be the same as [this problem](https://github.com/voxeltycoon/issues/discussions/630).

**Comment by andrewpey** [comment, marked as answer]
We've pushed a fix to the [beta branch](https://voxeltycoon.xyz/devlog/beta-branch/), please give it a try.
